styles/all/template/event/forumlist_body_category_header_before.html

```diff
@@ -0,0 +1,7 @@
+<!-- IF forumrow.S_IS_CAT -->
```

Comment:
From what I can see, the current implementation appears to work with subforums in viewforum.php